svc-api/handle/chassis.go

```diff
@@ -167,10 +167,12 @@ func (chassis *ChassisRPCs) GetChassis(ctx iris.Context) {
 		ctx.JSON(&response.Body)
 		return
 	}
-
+	requestData := strings.Split(req.RequestParam, ":")
```

Comment:
This code should come in svc-systems and not in svc-api.. svc-api is majorly for routing request and responses no specific business logic should be placed here

Reply:
I had tried doing changes in svs-systems/chassis/getresource.go but it didn't work from there...so here just for the display purpose I modified the response just before display.. would discuss this with team